.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**PR Buddy:** _@ mention a team member_
+
```

Comment:
I would suggest "PR Buddy" being automatic: the person responds to the review will be assumed as buddy

Reply:
I don't believe this can be automated. It might be best if whoever becomes the buddy inserts their own handle here?

Reply:
I think as soon as someone responds, the large reviewers list is removed. You can see the example in this PR. Right now only my name is up there

Reply:
Never mind. It appears to be inconsistent. Other PRs have all the people back on again even there were some reviews. Anyway, I don't think introducing "PR buddy" is correct workflow. There are 3 roles for a PR: author(s), reviewer(s) and tester(s). CI is one of the testers. We should not expect reviewers assume new "buddy" role to test PRs. That will be too much burden for reviewers. Test may need dedicated setup and complicated procedures.

Reply:
I will shift this away from the role of "PR buddy" and toward defining the responsibilities of the approver and tester.

Reply:
Since there are so many SoCs, platforms and use cases, unless some exceptional cases, CI should be the only tester. It will be a huge waste of resources to require anyone to manually execute these huge list of testing. We should spend effort to enhance CI testing instead. Engineering resource should be used to improve and debugg CI testing failures.